ui/src/elements/ngm-cam-coordinates.ts

```diff
@@ -47,7 +115,12 @@ export class NgmCamCoordinates extends LitElementI18n {
           </div>
         </div>
         ${i18next.t('camera_position_coordinates_label')}
-        <label class="ngm-coords user-select-text">${c[0]}, ${c[1]}</label>
+        <input class="ngm-coords ngm-coords-input"
+                 type="text"
+                 .value=${c[0] + ', ' + c[1]}
+                 @blur="${this.updateCoordinates}"
+                 @keydown="${(e) => { if (e.key === 'Enter') this.updateCoordinates(); }}"
```

Comment:
This could be written as an expression - not a strict requirement, I just prefer this over inline `if` statements.
```suggestion
                 @keydown="${(e) => key === 'Enter' && this.updateCoordinates()}"
```